kernel/src/driver.rs

```diff
@@ -184,7 +185,13 @@ pub trait Driver {
     /// is signaled with an upcall). Command 0 is a reserved command to
     /// detect if a peripheral system call driver is installed and must
     /// always return a CommandReturn::Success.
-    fn command(&self, which: usize, r2: usize, r3: usize, caller_id: ProcessId) -> CommandReturn {
+    fn command(
+        &self,
+        command_num: usize,
+        r2: usize,
+        r3: usize,
+        caller_id: ProcessId,
```

Comment:
Can we call this process_id

(I can make the change. I don't know whose court the ball is in since you pushed the change for me; just let me know)

Reply:
Yes good catch.